### PR TITLE
Support Windows colors

### PR DESCRIPTION
**Pull request by mattn**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our [guidelines](https://github.com/get-woke/woke/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Support Windows colors

**What is the current behavior?** (You can also link to an open issue here)

image

**What is the new behavior (if this is a feature change)?**

image

**Does this PR introduce a breaking change?** (What changes might users need to make due to this PR?)

No

**Other information**:

No

### CODE REVIEW

**Comment by mattn** [comment]
One of CIs is failing but seems not be caused by this change.

**Comment by mattn** [comment]
Please wait merge. I found another to fix.

**Comment by caitlinelfring** [comment]
Thanks for the contribution! It looks like it might be an issue with the `captureOutput` function https://github.com/get-woke/woke/blob/main/pkg/printer/util_test.go#L44 now that `color.Output` is used. I'll stand by

**Comment by mattn** [comment]
Please use squash merge if you feel noisy commits. Thanks.

File: cmd/root.go [hunk 1]
```diff
@@ -146,6 +147,9 @@ func setLogLevel() {
 	if debug {
 		zerolog.SetGlobalLevel(zerolog.DebugLevel)
 	}
+	if runtime.GOOS == "windows" {
+		log.Output(zerolog.ConsoleWriter{Out: color.Output})
+	}
```
Comment (mattn) [inline cmd/root.go:152]: Do you think that this should move to main.go ?

Comment (caitlinelfring) [reply cmd/root.go:152]: I'm a little confused by this should only apply to windows. If I'm reading correctly, [`color.Output`](https://github.com/fatih/color/blob/master/color.go#L25) = `colorable.NewColorableStdout()`, which should be Stdout regardless of OS right? Same comment for https://github.com/get-woke/woke/pull/20/files#diff-7f2a90cc6cb2d275be21c4c36d18055821064a64d15b3fccd53e1641c77a03cdR33-R35

Comment (mattn) [reply cmd/root.go]: I'll use mattn/go-colorable directly. Thanks.

Comment (caitlinelfring) [reply cmd/root.go]: Works for me, thanks! Appreciate the contribution... it made me realize I should be running CI tests against windows as well so I'll be looking into that.

**Review by caitlinelfring — APPROVED** [review]

**Comment by mattn** [comment]
Thank you. Sorry about OT, but I can't find a way for question. Can I change behavior of default rules? For example disabling `dummy`.

**Comment by caitlinelfring** [comment]
Yes there is, I realize I don't have it documented, so will update the README.

In your `.woke.yml` file, you can disable a rule by defining it with no `terms` or `alternatives`, so:

```yml
rules:
  - name: dummy
```

**Comment by mattn** [comment]
Yes, I figured out the way from reading codes. But if I set only `-name: dummy` line, rules that have not been detected until the change are displayed. (mainly file name check)

**Comment by caitlinelfring** [comment]
There might be bugs running on windows, I haven't tried running it on windows before today and the tests are returning some strange behavior that I'm not sure about (https://github.com/get-woke/woke/pull/21). Would you mind filing a GitHub issue with all the details?

**Pull request merged**